cfme/tests/automate/test_service_automate.py

```diff
@@ -0,0 +1,142 @@
+from cfme.provisioning import do_vm_provisioning
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.blockers import BZ
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log_validator import LogValidator
+
+
+pytestmark = [test_requirements.automate]
+
+
+@pytest.fixture(scope="module")
+def new_users(appliance):
+    """This fixture creates new users"""
+    users = []
+    for i in range(2):
+        user = appliance.collections.users.create(
+            name="user_{}".format(fauxfactory.gen_alphanumeric().lower()),
+            credential=Credential(principal='uid{}'.format(fauxfactory.gen_alphanumeric(4)),
+                                  secret=fauxfactory.gen_alphanumeric(4)),
+            email=fauxfactory.gen_email(),
+            groups=appliance.collections.groups.instantiate(
+                description="EvmGroup-super_administrator"),
+            cost_center="Workload",
+            value_assign="Database",
+        )
+        users.append(user)
+    yield users
+    if not BZ(1720273).blocks and user in users:
+        user.delete_if_exists()
+
+
+@pytest.fixture(scope='function')
+def infra_validate_request(domain):
+    # Take the 'ProvisionRequestApproval' class and copy it for own purpose.
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="Infrastructure"
+    ).namespaces.instantiate(name="VM").namespaces.instantiate(
+        name="Provisioning").namespaces.instantiate(name="StateMachines").classes.instantiate(
+        name="ProvisionRequestApproval").methods.instantiate(
+        name="validate_request").copy_to(domain.name)
+
+    method = domain.namespaces.instantiate(
+        name="Infrastructure"
+    ).namespaces.instantiate(name="VM").namespaces.instantiate(
+        name="Provisioning").namespaces.instantiate(name="StateMachines").classes.instantiate(
+        name="ProvisionRequestApproval").methods.instantiate(name="validate_request")
+    return method
+
+
+@pytest.fixture(scope='function')
+def service_validate_request(domain):
+    # Take the 'ProvisionRequestApproval' class and copy it for own purpose.
+    domain.parent.instantiate(name="ManageIQ").namespaces.instantiate(
+        name="Service"
+    ).namespaces.instantiate(name="Provisioning").namespaces.instantiate(
+        name="StateMachines").classes.instantiate(
+        name="ServiceProvisionRequestApproval").methods.instantiate(
+        name="validate_request").copy_to(domain.name)
+
+    method = domain.namespaces.instantiate(
+        name="Service"
+    ).namespaces.instantiate(name="Provisioning").namespaces.instantiate(
+        name="StateMachines").classes.instantiate(
+        name="ServiceProvisionRequestApproval").methods.instantiate(
+        name="validate_request")
+    return method
+
+
+@pytest.mark.tier(3)
+@pytest.mark.provider([VMwareProvider], scope="module")
+@pytest.mark.meta(blockers=[BZ(1671563, forced_streams=['5.10'])])
```

Comment:
@ganeshhubale is BZ 1671563 going to be backported to 5.10? If not, you can just ignore the stream.

Reply:
Actually, BZ was reported on 5.9z but fixed in 5.11 only.
Hence using ignore stream.